### PR TITLE
update iptables=false documentation

### PR DESCRIPTION
**Pull request by adamancini**
make it clearer that this is not intended for end-users

### CODE REVIEW

**Comment by arkodg** [comment]
@adamancini  is this comment `It is not possible to completely prevent Docker from creating iptables rules` referring to the `DOCKER-INGRESS` chain that is created even when `iptables=false`

**Comment by adamancini** [comment]
@arkodg I don't remember, specifically, but this comes up in discussion with @programmerq on the community IRC often enough that I thought it was worth calling out.

**Comment by programmerq** [comment]
@arkodg I would need to do some digging for the specific reference, but particularly swarm mode will ignore `--iptables=false`. I haven't personally checked it in several versions of Docker. The discussion I remember having about it was likely in the docker community slack team and is old enough that it wouldn't be visible.

tldr; check if swarm mode still ignores it too.

**Comment by adamancini** [comment]
```
docker@ada-testkit-EE35EF-ubuntu-0:~$ sudo su -
root@ada-testkit-EE35EF-ubuntu-0:~# cd /etc/docker/
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# vim daemon.json
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# iptables -F
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# iptables -t nat -F
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# iptables -t mangle -F
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# systemctl restart docker
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# iptables-save
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:02:31 2019
*mangle
:PREROUTING ACCEPT [74:5432]
:INPUT ACCEPT [74:5432]
:FORWARD ACCEPT [0:0]
:OUTPUT ACCEPT [49:4332]
:POSTROUTING ACCEPT [49:4332]
COMMIT
# Completed on Tue Sep 24 17:02:31 2019
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:02:31 2019
*nat
:PREROUTING ACCEPT [0:0]
:INPUT ACCEPT [0:0]
:OUTPUT ACCEPT [0:0]
:POSTROUTING ACCEPT [0:0]
:DOCKER - [0:0]
COMMIT
# Completed on Tue Sep 24 17:02:31 2019
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:02:31 2019
*filter
:INPUT ACCEPT [162:11744]
:FORWARD DROP [0:0]
:OUTPUT ACCEPT [108:10036]
:DOCKER - [0:0]
:DOCKER-ISOLATION-STAGE-1 - [0:0]
:DOCKER-ISOLATION-STAGE-2 - [0:0]
COMMIT
# Completed on Tue Sep 24 17:02:31 2019
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# docker service create -P nginx
unknown shorthand flag: 'P' in -P
See 'docker service create --help'.
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# docker service create -p 80 nginx
wi1k8cjequkia5kd2s0dp6m5n
overall progress: 0 out of 1 tasks
1/1: starting  [============================================>      ]
^COperation continuing in background.
Use `docker service ps wi1k8cjequkia5kd2s0dp6m5n` to check progress.
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker# iptables-save
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:03:03 2019
*mangle
:PREROUTING ACCEPT [9197:51326043]
:INPUT ACCEPT [9197:51326043]
:FORWARD ACCEPT [0:0]
:OUTPUT ACCEPT [7425:368082]
:POSTROUTING ACCEPT [7425:368082]
COMMIT
# Completed on Tue Sep 24 17:03:03 2019
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:03:03 2019
*nat
:PREROUTING ACCEPT [0:0]
:INPUT ACCEPT [0:0]
:OUTPUT ACCEPT [0:0]
:POSTROUTING ACCEPT [0:0]
:DOCKER - [0:0]
:DOCKER-INGRESS - [0:0]
-A PREROUTING -m addrtype --dst-type LOCAL -j DOCKER-INGRESS
-A OUTPUT -m addrtype --dst-type LOCAL -j DOCKER-INGRESS
-A POSTROUTING -o docker_gwbridge -m addrtype --src-type LOCAL -j MASQUERADE
-A DOCKER-INGRESS -p tcp -m tcp --dport 30000 -j DNAT --to-destination 172.18.0.2:30000
-A DOCKER-INGRESS -j RETURN
COMMIT
# Completed on Tue Sep 24 17:03:03 2019
# Generated by iptables-save v1.6.1 on Tue Sep 24 17:03:03 2019
*filter
:INPUT ACCEPT [28:2056]
:FORWARD DROP [0:0]
:OUTPUT ACCEPT [22:2596]
:DOCKER - [0:0]
:DOCKER-INGRESS - [0:0]
:DOCKER-ISOLATION-STAGE-1 - [0:0]
:DOCKER-ISOLATION-STAGE-2 - [0:0]
:DOCKER-USER - [0:0]
-A FORWARD -j DOCKER-USER
-A FORWARD -j DOCKER-INGRESS
-A DOCKER-INGRESS -p tcp -m tcp --dport 30000 -j ACCEPT
-A DOCKER-INGRESS -p tcp -m state --state RELATED,ESTABLISHED -m tcp --sport 30000 -j ACCEPT
-A DOCKER-INGRESS -j RETURN
-A DOCKER-USER -j RETURN
COMMIT
# Completed on Tue Sep 24 17:03:03 2019
root@ada-testkit-EE35EF-ubuntu-0:/etc/docker#
```

**Comment by arkodg** [comment]
@programmerq in the last release we removed the `DOCKER-USER` chain for the above option 
 thanks for sharing the rules in this thread @adamancini, there's a issue raised in `libnetwork` to highlight it (DOCKER-INGRESS needs to be disabled as well for the above case) https://github.com/docker/libnetwork/issues/2375

I'm okay approving this PR

**Comment by usha-mandya** [comment]
Thanks for the update.
